cli/dcoscli/data/help/cluster.txt

```diff
@@ -50,6 +50,8 @@ Options:
         Specify the authentication provider to use for login.
     --private-key=<key_path>
         Specify the path to a file that contains the private key.
+    -q, --quiet
+        Only display cluster IDs
```

Comment:
Here we refer to `IDs`, but it seems as if we refer to them as `names` elsewhere.